src/libraries/System.Runtime/tests/System/Reflection/NullabilityInfoContextTests.cs

```diff
@@ -739,7 +741,7 @@ public void NullablePublicOnlyOtherTypesTest()
             PropertyInfo publicGetPrivateSetNullableProperty = typeof(FileSystemEntry).GetProperty("Directory", flags)!;
             info = nullabilityContext.Create(publicGetPrivateSetNullableProperty);
             Assert.Equal(NullabilityState.NotNull, info.ReadState);
-            Assert.Equal(NullabilityState.Unknown, info.WriteState);
+            Assert.Equal(NullabilityState.NotNull, info.WriteState);
```

Comment:
Should be NotNull because the property is a non-nullable value type. Lack of nullable attributes doesn't matter.